### ISSUE TITLE
"Enqueue" config ignoring "disable_indexing" config

### ISSUE THREAD

**Issue by yogodoshi** [algolia/algoliasearch-rails#173]
Using `1.16.3` version.

```
algoliasearch enqueue: :reindex_with_sidekiq, per_environment: true, disable_indexing: Rails.env.test? do
...
end
```

The `reindex_with_sidekiq` is still being called on my specs.

ps: happy new year 🎉

**Comment by redox** [comment]
Good catch @yogodoshi; I'll wait Travis's go and release a new patch version of the gem 👍

**Comment by yogodoshi** [comment]
Wow, that was very fast! Thanks!

**Comment by redox** [comment]
This has been published in `1.17.0` @yogodoshi
